packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -101,6 +104,26 @@ function buildFilters(
   }
 }
 
+function removeRelationships(
+  rowId: string,
+  table: Table,
+  isManyToMany: boolean,
+  colName?: string
+) {
+  const tableId = table._id!
+  const filters = buildFilters(rowId, {}, table)
+  // safety check, if there are no filters on deletion bad things happen
+  if (Object.keys(filters).length !== 0) {
+    const op = isManyToMany ? Operation.DELETE : Operation.UPDATE
+    const body = colName && !isManyToMany ? { [colName]: null } : undefined
```

Comment:
It may make for cleaner code to split this out into 2 functions, one for many-to-many and one for, presumably, one-to-many / many-to-one?